src/server_manager/web_app/ui_components/outline-gcp-create-server-app.ts

```diff
@@ -303,32 +307,55 @@ export class GcpCreateServerApp extends LitElement {
     } else {
       if (!this.billingAccounts || this.billingAccounts.length === 0) {
         this.showBillingAccountSetup();
+        // Check every five seconds to see if an account has been added.
+        this.billingAccountsRefreshLoop = window.setInterval(() => {
+          this.refreshBillingAccounts();
+        }, 5000);
       } else {
         this.showProjectSetup(this.project);
       }
     }
   }
 
+  disconnectedCallback() {
+    super.disconnectedCallback();
+    this.stopRefreshingBillingAccounts();
+  }
+
   private init() {
     this.currentPage = '';
     this.selectedProjectId = '';
     this.selectedBillingAccountId = '';
+    this.stopRefreshingBillingAccounts();
   }
 
   private showBillingAccountSetup(): void {
     this.currentPage = 'billingAccountSetup';
   }
 
-  private async handleBillingVerificationNextTap(): Promise<void> {
-    this.showProjectSetup();
-  }
-
-  private async showProjectSetup(existingProject?: Project): Promise<void> {
+  private async refreshBillingAccounts(): Promise<void> {
     this.billingAccounts = await this.account.listBillingAccounts();
-    if (!this.billingAccounts || this.billingAccounts.length === 0) {
-      return this.showBillingAccountSetup();
+    // TODO: listBillingAccounts() can reject, resulting in an uncaught
+    // exception here that is shown in the debug console but not reflected
+    // in the UI.  We need to something better than failing silently.
+
+    if (this.billingAccounts && this.billingAccounts.length > 0) {
+      this.stopRefreshingBillingAccounts();
+      this.showProjectSetup();
+      window.bringToFront();
     }
+  }
+
+  private stopRefreshingBillingAccounts(): void {
+    window.clearInterval(this.billingAccountsRefreshLoop);
+    this.billingAccountsRefreshLoop = null;
+  }
 
+  private openBillingPage(): void {
+    window.open("https://console.cloud.google.com/billing");
```

Comment:
How do we open urls in other places of the code?

Reply:
This is similar to how it's done in browser_main.ts.  I changed this one to make them more similar (and fixed an error in the other one).